app/views/corporate/extra/_tool.erb

```diff
@@ -0,0 +1,74 @@
+<%= form_for(@partner, class: 'form t-embed-form',
+                          data: { dough_component: 'EmbedCodeGenerator Validation' },
+                          html: { id: 'embed_code', novalidate: 'noValidate', },
+                          url: {action: 'create'},
+                          builder: Dough::Forms::Builders::Validation) do |f| %>
+
+  <%= f.validation_summary %>
+  <%= hidden_field_tag :id, article.id %>
+
+  <%= f.form_row :name do %>
+    <%= f.errors_for :name %>
+    <%= f.label :name, t('tool_page.form.name_label'), class: 'form__label-heading' %>
+    <%= f.text_field :name, class: 't-organisation form__input',
+    data: { dough_validation_empty: 'You must enter an organisation' }, required: true, 'aria-required' => 'true', 'aria-invalid' => 'true' %>
```

Comment:
Under normal circumstances yup but we've been told that there is no need to support Welsh for these pages/sections.